pkg/oc/cli/experimental/config/patch.go

```diff
@@ -50,8 +50,15 @@ var (
 		%[1]s openshift.local.config/master/master-config.yaml --patch='{"auditConfig": {"enabled": true}}'`)
 )
 
+func NewPatchOptions(streams genericclioptions.IOStreams) *PatchOptions {
+	return &PatchOptions{
+		PrintFlags: genericclioptions.NewPrintFlags("patched").WithTypeSetter(configapi.Scheme).WithDefaultOutput("yaml"),
```

Comment:
@deads2k I'm unfamiliar with this scheme, but it is the one this command uses with the resource builder so I am using it here as well

Reply:
Yes, this scheme has only the config types, which is the only types needed for this command to know.